packages/client/src/client.js

```diff
@@ -188,15 +190,24 @@ export class PercyClient {
     return this.get(`projects/${project}/builds?${qs}`);
   }
 
+  // required so we could mock this method in tests :P
+  isIntervalLessThanThreshold(interval) {
+    return interval < MIN_POLLING_INTERVAL;
+  }
+
```

Comment:
We're not using this in tests now, so can this be removed?

Reply:
removed